gradle.properties

```diff
@@ -24,7 +24,7 @@ POM_SNAPSHOT_VERSION_NAME=2.1.0-SNAPSHOT
 # The setting is particularly useful for tweaking memory settings.
 org.gradle.jvmargs=-Xmx2048M
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
[Docs](https://developer.android.com/jetpack/androidx#using_androidx_libraries_in_your_project):
`android.enableJetifier`: When this flag is set to true, the Android plugin automatically migrates existing third-party libraries to use AndroidX dependencies by rewriting their binaries. **The flag is false by default if it is not specified.**
```suggestion
```